docker-compose.deploy.yml

```diff
@@ -1,15 +1,15 @@
 version: '3.4'
 
 services:
-  app:
-    environment:
-      DJANGO_MANAGEPY_MIGRATE: "on"
+  django:
+    build:
+      context: .
+      target: deploy
     env_file:
-      - .envs/.production/.django
-    image: petition:latest
-    links:
-      - db:db
-      - redis:redis
+      - .envs/.production-local-testing/.django
```

Comment:
This env file name seems off to me. Does the actual deploy ignore/omit this env file? Is there a way to make sure we don't include such a file in the deploy?

Reply:
Heroku and the production deploy don't use this file. It's only used to test building and running the `deploy` stage of the Dockerfile.

And yes, this file isn't committed and is in gitignore:

https://github.com/deardurham/dear-petition/blob/5f2bce75d5efc22761ffa07de0aaa076777aa3dd/.gitignore#L287-L288